### PR TITLE
Updated configuration

### PR DESCRIPTION
**Pull request by antoniocarlon**
Updated the User database configuration section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Steps to deploy a new Data Services API version :
      ```
    - Give permission to execute and select to the `dataservices_user` user:
      ```
-     psql -U postgres -d dataservices_db -c "BEGIN;CREATE EXTENSION IF NOT EXISTS observatory; COMMIT" -e
+     psql -U postgres -d dataservices_db -c "BEGIN;CREATE EXTENSION IF NOT EXISTS observatory version 'dev'; COMMIT" -e
```
Comment (javitonino) [inline README.md]: Nitpicky, but can you make `version` uppercase? For consistency with the rest of SQL queries.

File: README.md [hunk 2]
```diff
@@ -254,14 +254,16 @@ jsonb_set(
 ```
 ### User database configuration
 
+#### Option 1
```
Comment (javitonino) [inline README.md]: Maybe `#### Option 1 (manually)`

File: README.md [hunk 3]
```diff
@@ -272,10 +274,31 @@ SELECT CDB_Conf_SetConf(
...
-#### Search path
+##### Search path
 
 The search path must be configured in order to be able to execute the functions without using the schema:
 
 ```sql
 ALTER ROLE "<USER_ROLE>" SET search_path="$user", public, cartodb, cdb_dataservices_client;
 ```
+
+#### Option 2
```
Comment (javitonino) [inline README.md]: Maybe `#### Option 2 (from Builder)`

File: README.md [hunk 4]
```diff
@@ -272,10 +274,31 @@ SELECT CDB_Conf_SetConf(
...
+
+In the `cartodb/config/app_config.yml` file, enable the desired dataservices:
+```
+enabled:
+    geocoder_internal: false
+    hires_geocoder: false
+    isolines: false
+    routing: false
+    data_observatory: true
+```
```
Comment (javitonino) [inline README.md]: I think we are missing a rake execution, aren't we?

**Comment by javitonino** [comment]
@ethervoid Mind taking a final look as out local dataservices expert? 🙏

**Comment by ethervoid** [comment]
I don't think the option 2 should be included in the public readme. In that case, we have to include in the installation steps but we shouldn't mix the DS readme (which shouldn't know about other projects) with the installation documentation

**Comment by javitonino** [comment]
Well, we need to document this somehow. We could add it to the installation guide http://cartodb.readthedocs.io/en/latest/operations/configure_data_services.html if that sounds better to you. I'd still think it'd be nice to to link it from here though.

We can pretend that DS API is independent from CARTO, but not really. But I agree that separating it to the installation guide is good, this README is pretty long already.

**Pull request merged**